src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160512_status_change_log.xml

```diff
@@ -0,0 +1,92 @@
+
+        <createTable tableName="STAT_WORKFLOW_STATUS">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="workflow_id" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="timestamp" type="TIMESTAMP(6)" defaultValueDate="2001-01-01 01:01:01.0">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createIndex indexName="IDX_STAT_WORKFLOW_STATUS_WORKFLOW_ID" tableName="STAT_WORKFLOW_STATUS" unique="false">
+            <column name="workflow_id"/>
+        </createIndex>
+
+        <createTable tableName="STAT_SUBMISSION_STATUS">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="submission_id" type="BINARY(16)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="timestamp" type="TIMESTAMP(6)" defaultValueDate="2001-01-01 01:01:01.0">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createIndex indexName="IDX_STAT_SUBMISSION_STATUS_SUBMISSION_ID" tableName="STAT_SUBMISSION_STATUS" unique="false">
+            <column name="submission_id"/>
+        </createIndex>
+
+    </changeSet>
+
+
+    <changeSet logicalFilePath="dummy" author="davidan" id="status_change_log_triggers">
+        <sql>
+            DROP TRIGGER IF EXISTS after_workflow_insert;
+            DROP TRIGGER IF EXISTS after_workflow_update;
+            DROP TRIGGER IF EXISTS after_submission_insert;
+            DROP TRIGGER IF EXISTS after_submission_update;
+
+            CREATE TRIGGER after_workflow_insert AFTER INSERT ON WORKFLOW
+                FOR EACH ROW
+                    INSERT INTO STAT_WORKFLOW_STATUS
+                        (workflow_id, status, timestamp)
+                    VALUES
+                        (NEW.ID, NEW.STATUS, NEW.STATUS_LAST_CHANGED);
+
+
+            CREATE TRIGGER after_workflow_update AFTER UPDATE ON WORKFLOW
+                FOR EACH ROW
+                    INSERT INTO STAT_WORKFLOW_STATUS
+                        (workflow_id, status, timestamp)
+                    VALUES
+                        (NEW.ID, NEW.STATUS, NEW.STATUS_LAST_CHANGED);
+
+            CREATE TRIGGER after_submission_insert AFTER INSERT ON SUBMISSION
+                FOR EACH ROW
+                    INSERT INTO STAT_SUBMISSION_STATUS
+                        (submission_id, status, timestamp)
+                    VALUES
+                        (NEW.ID, NEW.STATUS, CURRENT_TIMESTAMP);
```

Comment:
for workflows, we can use the timestamp provided by Scala. For submissions, we don't have one of these, so we rely on the database's timestamp. I waffled on this: is it better to be consistent between the two STAT_* tables, always using db's timestamp? Or is it better to be consistent between WORKFLOW and STAT_WORKFLOW_STATUS, using the exact same value for each?